README.md

```diff
@@ -35,84 +35,56 @@ import "github.com/swaggo/files" // swagger embed files
 ```
 
 ### Canonical example:
-
+Now assume you have implemented a simple api as following:
 ```go
-package main
-
-import (
-	"github.com/gin-gonic/gin"
-	swaggerFiles "github.com/swaggo/files"
-	ginSwagger "github.com/swaggo/gin-swagger"
-
-	_ "github.com/swaggo/gin-swagger/example/basic/docs" // docs is generated by Swag CLI, you have to import it.
-)
+//A get function which returns a Helloworld for you 
+func Helloworld(g *gin.Context)  {
+	g.JSON(http.StatusOK,"helloworld")
+}
 
-// @title Swagger Example API
-// @version 1.0
-// @description This is a sample server Petstore server.
-// @termsOfService http://swagger.io/terms/
+```
+So how to use gin-swagger on api above? Just follow the folling guide:)  
 
-// @contact.name API Support
-// @contact.url http://www.swagger.io/support
-// @contact.email support@swagger.io
+1. Add Comments for apis and main function with gin-swagger rules like following:  
+```go
+// PingExample godoc
+// @Summary ping example
+// @Schemes
+// @Description do ping
+// @Tags example
+// @Accept json
+// @Produce json
+// @Success 200 {string} Helloworld
+// @Router /example/helloworld [get]
+func Helloworld(g *gin.Context)  {
+	g.JSON(http.StatusOK,"helloworld")
+}
 
-// @license.name Apache 2.0
-// @license.url http://www.apache.org/licenses/LICENSE-2.0.html
 
-// @host petstore.swagger.io
-// @BasePath /v2
+//Comments used to generate swagger docs are igonred
+//We foucs on how to setup gin router in main
 func main() {
-	r := gin.New()
-
-	url := ginSwagger.URL("http://localhost:8080/swagger/doc.json") // The url pointing to API definition
-	r.GET("/swagger/*any", ginSwagger.WrapHandler(swaggerFiles.Handler, url))
-
-	r.Run()
+	r := gin.Default()
+
+    //Using this 
+	v1 := r.Group("/api/v1")
```

Comment:
I've tested this example just now, and when I tried to execute it from http://localhost:8080/swagger/index.html#/example/get_example_helloworld, it returned a 404 error because docs.SwaggerInfo.BasePath in not include "/api/v1". It works if I modify it like this, using a hit from https://github.com/swaggo/swag#how-to-use-it-with-gin:
```diff
diff --git a/README.md b/README.md
index b9b495c..4bbe761 100644
--- a/README.md
+++ b/README.md
@@ -68,4 +68,5 @@ func main() {
 
     //Using this 
+    docs.SwaggerInfo.BasePath = "/api/v1"
 	v1 := r.Group("/api/v1")
 	{
@@ -85,5 +86,5 @@ Assume your project name is `MyGoSwagger`,don't forget the blank identifier!
 ```go
 import (
-    _ "MyGoSwagger/docs"
+    docs "MyGoSwagger/docs"
 )
 ```
 ```go
 import (
-    _ "MyGoSwagger/docs"
+    docs "MyGoSwagger/docs"
 )
 ```
```

Reply:
So sorry for ignoring this message so long. It's lost at my timeline. I will fix it soon:)
Thanks for reviewing.